po/de.po

```diff
@@ -22251,9 +22251,8 @@ msgid "Have a nice day!"
 msgstr "Einen schönen Tag noch."
 
 #: gnucash/python/init.py:118
-#, fuzzy, python-format
```

Comment:
Remove `, fuzzy`, but not `, python-format`. Otherwise it would really display "Willkommen zur GnuCash %s Shell" instead of the parameter.

Fixed in my following commit.

Something else: use in the future more descriptive commit messages, e.g. in this case by using the prefix `L12N:de: `